ckanext/dataportaltheme/fanstatic/githubfeed.css

```diff
@@ -0,0 +1,23 @@
+body {
```

Comment:
This style is already defined in [`dataportaltheme.css`](https://github.com/code4romania/ckanext-dataportaltheme/blob/master/ckanext/dataportaltheme/fanstatic/dataportaltheme.css#L3)